httpobs/scanner/analyzer/headers.py

```diff
@@ -542,3 +542,61 @@ def x_xss_protection(reqs: dict, expectation='x-xss-protection-1-mode-block') ->
             output['result'] = 'x-xss-protection-not-needed-due-to-csp'
 
     return output
+
+
+@scored_test
+def http_referrer_policy(reqs: dict, expectation='referrer-header-enabled') -> dict:
+    """
+    :param reqs: dictionary containing all the request and response objects
+    :param expectation: test expectation
+        referrer-policy-header-enabled: HTTP Referrer-Policy header enabled
```

Comment:
I like to be a bit more explicit about things than just "-enabled"; such as talking about what the values are.